collect_app/src/main/java/org/odk/collect/android/injection/config/AppDependencyComponent.java

```diff
@@ -261,6 +262,8 @@ interface Builder {
 
     void inject(FormHierarchyFragmentHostActivity formHierarchyFragmentHostActivity);
 
+    void inject(WidgetAnswerTextView widgetAnswerText);
```

Comment:
I'd love to avoid using Dagger here. Could we add a `setFontSize` to `WidgetAnswerView` instead?

Reply:
Ok.